recursio-client/Managers/CharacterManager.gd

```diff
@@ -192,11 +192,21 @@ func _on_player_timeline_changed(timeline_index) -> void:
 	_player.move_to_spawn_point()
 	_ghost_manager.refresh_active_ghosts()
 	_ghost_manager.refresh_path_select()
+	
+	if _round_manager.get_current_phase() == RoundManager.Phases.PREPARATION:
```

Comment:
I think I realize why this check is here, but it seems kind of redundant at first glance, so I think it'd be good to add a comment explaining why it's here. People have been known to delete your code if it seems redundant :eyes:

Reply:
Fixed